### PR TITLE
Add two missing sku data to product context

### PR DESCRIPTION
**Pull request by josantana**
#### What is the purpose of this pull request?

Expose two missing sku data to product context, so we don't need to make unnecessary extra requests for such a simple thing. The intent is to present the complementName on product shelves; and also present when the name of the seller of that specific product. With this fix we can access this data from the context using `useProductSummary`.

#### Screenshots or example usage

Screen Shot 2020-09-22 at 11 35 23

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by kaisermann — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.71.0] - 2020-09-22
+### Added
+- Add new fields to the SKU from the `products` query.
```
Comment (kaisermann) [inline CHANGELOG.md:12]: Add which fields here please 😁

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (josantana) [reply CHANGELOG.md]: Done!

**Comment by josantana** [comment]
Can anyone help me to approve this, please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.71.0] - 2020-09-22
 ### Added
-- `itemsWithSimulation` query.
+- Add `complementName` and `sellerName` to the SKU on `products` query.
```
Comment (klzns) [inline CHANGELOG.md:12]:
```suggestion
- `itemsWithSimulation` query.
```

The changes to the changelog should be under the [Unreleased] section. Example:

```diff
 ## [Unreleased]
+### Added
+- Add `complementName` and `sellerName` to the SKU on `products` query.

 ## [0.71.0] - 2020-09-22
```

**Review by klzns — CHANGES_REQUESTED** [review]
Thanks for the pull request! Just a minor fix in the changelog entry

**Comment by klzns** [comment]
Fixed by https://github.com/vtex-apps/store-resources/pull/132

**Pull request closed without merging**